.github/workflows/upload_pypi_slint_compiler.yaml

```diff
@@ -4,62 +4,103 @@
 name: Upload slint-compiler to Python Package Index
 
 on:
-    workflow_dispatch:
-      inputs:
-        release:
-            type: boolean
-            default: false
-            required: false
-            description: "Release? If false, publish to test.pypi.org, if true, publish to pypi.org"
+  workflow_dispatch:
+    inputs:
+      release:
+        type: boolean
+        default: false
+        required: false
+        description: "Release? If false, publish to test.pypi.org, if true, publish to pypi.org"
 
 jobs:
-    publish-to-test-pypi:
-        if: ${{ github.event.inputs.release != 'true' }}
-        name: >-
-            Publish Python 🐍 distribution 📦 to Test PyPI
-        runs-on: ubuntu-latest
-        environment:
-            name: testpypi
-            url: https://test.pypi.org/p/slint-compiler
-        permissions:
-            id-token: write  # IMPORTANT: mandatory for trusted publishing
-        steps:
-            - uses: actions/checkout@v5
-            - name: Install uv
-              uses: astral-sh/setup-uv@v7
-            - name: Build
-              run: uv build
-              working-directory: api/python/compiler
-            - uses: actions/upload-artifact@v4
-              with:
-                name: python-package-distributions
-                path: api/python/compiler/dist/*
-            - name: Publish
-              run: uv publish --publish-url https://test.pypi.org/legacy/
-              working-directory: api/python/compiler
+  build-wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          [
+            ubuntu-latest,
+            ubuntu-24.04-arm,
```

Comment:
Same here, `ubuntu-22.04-arm`.

The main reason is that we'd like the binary to have the lowest glibc dependency possible, to run on as many machines as possible.